### ISSUE TITLE
Connecting duplicated blocks can result in the stack being executed

### ISSUE THREAD

**Issue by thisandagain** [LLK/scratch-gui#1307]
![untitled3](https://user-images.githubusercontent.com/747641/35123568-d2730176-fc70-11e7-8575-e20375b97982.gif)

**Comment by paulkaplan** [comment]
This appears to be due to the pointer-events: none; that GUI applies to the `.blocklyBlockDragSurface`, but I'm not exactly sure why, still investigating. Removing that line fixes the problem, but there is a comment about why that line is needed. https://github.com/LLK/scratch-gui/blob/develop/src/components/blocks/blocks.css#L83

/ht @picklesrus

**Comment by ErikMejerHansen** [comment]
Possible duplicate of: https://github.com/LLK/scratch-blocks/issues/1935

**Comment by paulkaplan** [comment]
@ErikMejerHansen yup can confirm this is the same issue. The issue does not appear in the blocks playground, it is actually a problem introduced by the GUI CSS, so I'm going to close that one as a duplicate of this.

**Comment by adroitwhiz** [comment]
This is happening again because #5078 was reverted

**Comment by griffpatch** [comment]
As you probably are all aware, but I just wanted to state it. The execution
occurs because you click to drop the duplicated block stack. If you are
attaching to the bottom of another stack then the scripts are not executed,
but if you drop between other blocks in a block stack then the stack is
executed. This is because your cursor is 'over' the block stack when you
click to place the duplicated stack, and are therefore clicking the stack
behind too. If we could just cancel the event bubbling it might all go away?
